services/src/main/java/org/fao/geonet/api/records/MetadataSharingApi.java

```diff
@@ -429,7 +429,7 @@ MetadataProcessingReport unpublish(
     @ResponseStatus(HttpStatus.CREATED)
     public
     @ResponseBody
-    MetadataProcessingReport share(
+    MetadataProcessingReport shares(
```

Comment:
```suggestion
    MetadataProcessingReport shareMultipleRecords(
```

To be consistent with the other 2 method names updated.